frontend/app/routes/_gcweb-app.$.tsx

```diff
@@ -7,15 +7,14 @@ import { getTypedI18nNamespaces } from '~/utils/locale-utils';
 
 const i18nNamespaces = getTypedI18nNamespaces('gcweb');
 
-export const loader = async ({ request }: LoaderFunctionArgs) => {
-  return json(
-    {
-      i18nNamespaces,
-      pageIdentifier: 'CDCP-0404',
-      pageTitleI18nKey: 'gcweb:not-found.page-title',
-    } as const satisfies LoaderFunctionData,
-    { status: 404 },
-  );
+export const handle = {
+  i18nNamespaces,
+  pageIdentifier: 'CDCP-0404',
+  pageTitleI18nKey: 'gcweb:not-found.page-title',
+} as const satisfies RouteHandleData;
+
+export const loader = async () => {
+  return json(null, { status: 404 });
 };
```

Comment:
```suggestion
export async function loader({ request }: LoaderFunctionArgs) {
  return json(null, { status: 404 });
};
```

Reply:
We're using arrow function in a coulpe of places and I fixed them. `request` object is not used in this case and it's why I removed it. I added it as you're suggested.

Reply:
It's not terrible to remove it, but I find it helps when debugging issues to have it available. The eslint rule that barfs when a function parameter is not used is turned off for the project (as it should be, imo) so having the parameter there isn't a problem.